pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -462,6 +463,7 @@ type provisioningMetadata struct {
 }
 
 type dashboardIdentity struct {
+	// Deprecated: use FolderUID instead
```

Comment:
i've talked about this with serge! basically adding FolderUID will be the next step for us. and these comments help us identify where the replacements are needed / prevent the linter from complaining

in the future tho for a similar type of effort i'll definitely use your suggestion tho because i think it's clearer!

Reply:
(not changing it atm tho for consistency's sake)

Reply:
That makes sense given that we'll be adding a field called `FolderUID` as a next step.